packages/backend/src/guards/firebase.auth.ts

```diff
@@ -8,6 +8,9 @@ import {
   Credential,
 } from 'firebase-admin/app';
 import { DecodedIdToken, getAuth } from 'firebase-admin/auth';
+import { UserStoreService } from 'src/db/user/userStore.service';
+import { UserModel } from 'src/db/user/user.schema';
```

Comment:
Remove this UserModel and UserStoreService import as they have not been used

Reply:
this is just a nit so you probably shouldn't block the merge for it

Reply:
Sorry, shud I make changes directly then?